src/main/kotlin/uk/gov/justice/digital/hmpps/incidentreporting/resource/ReportResource.kt

```diff
@@ -357,11 +359,6 @@ class ReportResource(
         description = "Data not found",
         content = [Content(mediaType = "application/json", schema = Schema(implementation = ErrorResponse::class))],
       ),
-      ApiResponse(
-        responseCode = "409",
-        description = "Incident report already exists",
```

Comment:
the request to create a new draft report does not give the caller an option to provide any ids so this cannot happen